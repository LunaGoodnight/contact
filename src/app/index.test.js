import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import "@testing-library/jest-dom";

test("loads and displays title", async () => {
  expect(1).toBe(1);
});
