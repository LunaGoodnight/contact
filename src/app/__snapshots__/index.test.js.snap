// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative bg-white rounded-lg shadow translate-y-[-50px] transition-all duration-300 ease-in-out p-4 sm:w-4/6 w-5/6"
>
  <div>
    test wrapper
  </div>
</div>
`;

exports[`renders modal wrapper correctly 1`] = `
<div
  className="relative bg-white rounded-lg shadow translate-y-[-50px] transition-all duration-300 ease-in-out p-4 sm:w-4/6 w-5/6"
>
  <div>
    test wrapper
  </div>
</div>
`;
